python/looker_sdk/sdk/api31/models.py

```diff
@@ -5323,6 +5323,181 @@ def __init__(self, *, inherits: Optional[bool] = None):
         self.inherits = inherits
 
 
+@attr.s(auto_attribs=True, kw_only=True)
```

Comment:
is this against 7.8?

something is goofed up for the python generated code: Any `Write` model should have an `__init__` method. Also, it seems suspicious to me that there are so many new write classes in `3.1` (across all sdks). Looks like "id" got made read-only for `ContentValidation` things - we should probably investigate and follow up with the owner...

Reply:
another suspicious thing: with all these new (improperly generated) Write classes, nothing changed in `methods.py`...

Reply:
Thanks, Joel. I'll look into this some and let you know. It's probably something to do with the dynamic type generation after the spec is loaded. I think I made `loadDynamicTypes()` a bit too dumb/eager.